go.mod

```diff
@@ -120,7 +120,7 @@ require (
 	k8s.io/kubectl v0.28.0-beta.0
 	k8s.io/utils v0.0.0-20230711102312-30195339c3c7
 	sigs.k8s.io/controller-runtime v0.15.1
-	sigs.k8s.io/gateway-api v0.8.0-rc1
+	sigs.k8s.io/gateway-api v0.8.0-rc1.0.20230816004229-ce5bf58ac9bf
```

Comment:
This is to get fix in https://github.com/kubernetes-sigs/gateway-api/pull/2310

@howardjohn Is this change OK, or should we just hold this PR until the release?